repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -4,112 +4,149 @@
 <#include "./macro/pageIndex.ftl">
 <#include "./macro/publicationHeader.ftl">
 <#include "./macro/sections/imageSection.ftl">
+<#include "../common/macro/pubsBreadcrumb.ftl">
 <#assign formatFileSize="uk.nhs.digital.ps.directives.FileSizeFormatterDirective"?new() />
 <@hst.setBundle basename="publicationsystem.labels,publicationsystem.headers"/>
 <#-- @ftlvariable name="publication" type="uk.nhs.digital.ps.beans.Publication" -->
 
+<#-- Add meta tags -->
+<#include "../common/macro/metaTags.ftl">
```

Comment:
FYI - Just a heads up: At the moment the pubs templates don't populate the meta tags correctly, because the document object is not defined. Our implementation uses `document.title` and `document.seosummary` to populate the meta tags for SEO. There is no action for RPS for now, we'll implement a solution once we're done with the template transformation tasks.